.github/workflows/build.yml

```diff
@@ -137,3 +137,31 @@ jobs:
             DBUILD_DATE=${{ steps.date.outputs.DATE }}
             DBUILD_REPO_URL=https://github.com/openebs/dynamic-localpv-provisioner
             DBUILD_SITE_URL=https://openebs.io
+
+  ansible:
```

Comment:
Could you use the name 'localpv-e2e' here.
For the openebs/provisioner-localpv image the workflow is named 'provisioner-localpv'
I feel it makes sense to use the 'localpv-e2e' name for the openebs/localpv-e2e image.